python/doc/examples/reliability_sensitivity/reliability/plot_form_explained.py

```diff
@@ -11,256 +11,307 @@
+graph_model.setXTitle(r"$x_1$")
+graph_model.setYTitle(r"$x_2$")
+graph_model.setTitle(r"Isolines of the model : $g$")
+view = otv.View(graph_model, square_axes=True)
 
 
 # %%
+# We consider  the univariate output variable :
+#
+# .. math::
+#
+#   Y = \model(\inputRV)
+#
 # We want to estimate the probability :math:`P_f` of the output variable to be greater than a prescribed threshold :math:`s=10` : this is the failure event.
-# This probability is simply expressed as an integral:
+# This probability is simply expressed for a continuous random vector :math:`\inputRV` as:
 #
 # .. math::
+#    :label: PfDef
 #
-#    P_f = \int_{\mathcal{D}} \mathbf{1}_{\mathcal{D}}(x) df_{X_1,X_2}(x)
+#    P_f = \Prob{Y \geq s} = \int_{\set{D}} \mathbf{1}_{\set{D}}(x) \pdf d\vect{x}
+#
+# where:
+#
+# .. math::
 #
-# where :math:`\mathcal{D} = \{ (x_1, x_2) \in [0,+\infty[ \times \mathbb{R} / x_1 x_2 \geq s \}` is the failure domain.
-# In the general case the probability density function :math:`f_{X_1,X_2}` and the domain of integration :math:`\mathcal{D}` are difficult to handle.
+#  \set{D} = \{ (x_1, x_2) \in [0,+\infty[ \times \mathbb{R} \, | \,  \model(x_1, x_2) \geq s \}
+#
+# is the failure domain and :math:`\inputMeasure` is the probability density function (PDF)
+# of :math:`\inputRV`.
 
 # %%
 # We first define RandomVector objects and the failure event associated to the output random variable.
-vectorX = ot.RandomVector(distX)
-vectorY = ot.CompositeRandomVector(f, vectorX)
+vector_X = ot.RandomVector(dist_X)
+vector_Y = ot.CompositeRandomVector(g, vector_X)
 s = 10.0
-event = ot.ThresholdEvent(vectorY, ot.Greater(), s)
+event = ot.ThresholdEvent(vector_Y, ot.Greater(), s)
 
 
 # %%
-# This event can easily be represented with a 1D curve as it is a branch of an hyperbole.
-# If :math:`y =  x_1 x_2 = 10.0`, then the boundary of the domain of failure is the curve :
+# The boundary of the failure domain can easily be represented as it is a branch of an hyperbole: the
+# boundary is the graph of the function defined from :math:`\Rset` into  :math:`\Rset` by:
+#
+# .. math::
+#    :label: defH
+#
+#    h : x_1 \mapsto x_2 = \frac{s}{x_1}
+#
+# The boundary of the failure domain is also the isoline of the model :math:`\model` associated to the
+# level :math:`s`:
 #
 # .. math::
 #
-#    h : x_1 \mapsto \frac{10.0}{x_1}
+#    \partial \set{D} =  \{(x_1, x_2)\, |\, \model(x_1, x_2) = s \}
 #
+# We can draw it with the `draw` method of the function :math:`\model`.
+
+# %%
+nb_points = 101
+graph_g = g.draw([0.0, -10.0], [20.0, 10.0], [nb_points] * 2)
+draw_boundary = graph_g.getDrawable(0)
+draw_boundary.setLevels([s])
+draw_boundary.setLegend(r'Boundary $\partial \mathcal{D}$')
+graph_g.setDrawables([draw_boundary])
+
+# %%
+texts = [r" $\mathcal{D} = \{(x_1, x_2)\, |\, g(x_1, x_2) \geq 10 \}$"]
+text_graph = ot.Text([[10.0, 3.0]], texts)
+text_graph.setTextSize(1)
+text_graph.setColor("black")
+graph_g.add(text_graph)
 
 # %%
```

Comment:
This new block is unnecessary: it is the remaining part of the previous Python block script.

Reply:
No! It includes the text about the failure domain...